### PR TITLE
Add RemoveUserFromList / RemoveMultipleUsersFromList

### PR DESCRIPTION
**Pull request by hem6**
Add following endpoints
- /lists/members/destroy.json
- /lists/members/destroy_all.json

### CODE REVIEW

File: lists.go [hunk 1]
```diff
@@ -42,6 +42,31 @@ func (a TwitterApi) AddMultipleUsersToList(screenNames []string, listID int64, v
 	return list, r.err
 }
 
+// RemoveUserFromList implements /lists/members/destroy.json
+func (a TwitterApi) RemoveUserFromList(screenName string, listID int64, v url.Values) (users []User, err error) {
+	v = cleanValues(v)
+	v.Set("list_id", strconv.FormatInt(listID, 10))
+	v.Set("screen_name", screenName)
+
+	var addUserToListResponse AddUserToListResponse
```
Comment (muesli) [inline lists.go]: Is this actually the correct response?

Comment (hem6) [reply lists.go]: Thank you for your review. It's my mistake.:disappointed_relieved:
`/lists/members/destroy.json` returns the list.
Fixed the return type of `RemoveUserFromList`.

**Comment by muesli** [comment]
Thanks, LGTM at first glance. A test case for it would be nice though.

**Comment by hem6** [comment]
Thank you. I Added some test cases.

**Comment by amitizle** [comment]
Any update on this PR? looking forward using this endpoint 🙂

**Pull request merged**